local-config.yml

```diff
@@ -107,3 +107,14 @@ routes:
   headers: []
   params: []
   body: []
+- method: POST
+  url: https://reqres.in/api/register
+  headers: []
+  params: []
+  body:
+  - name: ""
+    value: ""
+  - name: email
+    value: eve.holt@reqres.in
+  - name: password
+    value: pistol
```

Comment:
why not an array?

Reply:
If we do not opt to store this as an array, we would be converting the response each time to an iterable "name" -> "value"

Reply:
It is an array. Non-empty arrays in yaml are represented in form of:

```
a:
- para_1:
  para_2:
  ..
- para_3:
  para_4
```

that corresponds to `[{para_1, para_2}, {para_3, para_4}]`